app/scripts/components/common/catalog/catalog-content.tsx

```diff
@@ -108,11 +108,11 @@ function CatalogContent({
 
   useEffect(() => {
     if (!selectedFilters.length) {
-      onAction(CatalogActions.CLEAR);
-
-      if (!isSelectable) {
-        navigate(DATASETS_PATH);
-      }
+      onAction(CatalogActions.CLEAR_TAXONOMY);
+      // @NOTE: When is this used? Is this safe to remove?
+      // if (!isSelectable && !search) {
+      //   navigate(DATASETS_PATH);
```

Comment:
This was wiping search query parameter, so I commented this out. But I am not sure what this was trying to achieve. Do you see any problem removing this line @sandrahoang686 ?

Reply:
🤔 i cannot remember, I think its safe to remove but I can help QA again before its merged

Reply:
Hmm, I don't think we need it; I should have asked when porting the Data Catalog to the modal if I should remove it but missed it.